src/routers/alpha-router/gas-models/gas-model.ts

```diff
@@ -94,7 +94,7 @@ export type BuildOnChainGasModelFactoryType = {
   gasPriceWei: BigNumber;
   v3poolProvider: IV3PoolProvider;
   token: Token;
-  v2poolProvider: IV2PoolProvider;
+  v2poolProvider?: IV2PoolProvider;
```

Comment:
Is this removable?

Reply:
Removed in https://github.com/violetprotocol/mauve-smart-order-router/pull/32/commits/ad2362124cee95942aa53006fcc010cd8858c43f